packages/features/flags/features.repository.ts

```diff
@@ -58,6 +59,27 @@ export class FeaturesRepository implements IFeaturesRepository {
     }, {} as AppFlags);
   }
 
+  /**
+   * Gets all features enabled for a specific team in a map format.
+   * @param teamId - The ID of the team to get features for
+   * @returns Promise<{ [slug: string]: boolean } | null>
+   */
+  public async getTeamFeatures(teamId: number) {
+    const result = await kysely
+      .selectFrom("TeamFeatures")
+      .innerJoin("Feature", "Feature.slug", "TeamFeatures.featureId")
+      .select(["Feature.slug", "Feature.enabled"])
+      .where("TeamFeatures.teamId", "=", teamId)
+      .execute();
```

Comment:
So we ditching Prisma officially.

Reply:
Not sure just asked this in engineering. I don't see why we would keep using it when it produces shit query outputs

Reply:
A lot of this file is using raw sql anywhere so feels even more useful here